### DISCUSSION TITLE
Global Values

### DISCUSSION THREAD

**Discussion by untidymusic** [RomanKubiak/ctrlr#672] [General]
Hi, I’m hoping someone can point me in the right direction.

I’m confused about setting global values ie k0, k1 etc using a modulator.

setGlobal (0, modulatorValue)

This works for k0 but how do I set a global value for k1, k2 etc using modulators and can I do it in separate modulators?

Many thanks in advance,
Mark

**Comment by dnaldoog** [comment]
You can set them from here:


![](https://user-images.githubusercontent.com/76831683/215406929-f9c343d7-0c29-41f4-a82c-a351a63eb19b.png)

**Reply by untidymusic** [reply]
Thank you so much.
